core/src/test/java/nl/nn/adapterframework/filesystem/FileSystemActorTest.java

```diff
@@ -82,11 +82,36 @@ public void fileSystemActorTestConfigureBasic() throws Exception {
 
 	@Test
 	public void fileSystemActorTestConfigureNoAction() throws Exception {
-		thrown.expectMessage("action must be specified");
+		thrown.expectMessage("either attribute [action] or parameter [action] must be specified");
 		thrown.expectMessage("fake owner of FileSystemActor");
 		actor.configure(fileSystem,null,owner);
 	}
 
+	@Test
+	public void fileSystemActorEmptyParameterAction() throws Exception {
+		thrown.expectMessage("unable to resolve the value of parameter");
```

Comment:
Please add two more tests:
- to show that when the value of the parameter is empty, the configured action is taken as a default
- to show when both attribute and parameter action are set and valid, then the value of the parameter is used.